package.json

```diff
@@ -82,6 +82,7 @@
     "concurrently": "^3.4.0",
     "cookie": "^0.3.1",
     "css-loader": "^0.28.4",
+    "date-fns": "^1.29.0",
```

Comment:
should we also remove moment here?

Reply:
moment is still used on the back-end, but it is now out of the front-end.